services/QuillLessonsServer/nodemon.json.example

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "JWT_PUBLIC_KEY": "-----BEGIN PUBLIC KEY-----\n SOME LEGITIMATE PUBLIC KEY GOES HERE \n NOTE THAT IT SHOULD BE THE CORRESPONDING PUBLIC KEY OF THE PRIVATE KEY VALUE SET IN YOUR LOCAL LMS .env FILE IN THE KEY 'LESSONS_PRIVATE_KEY' \n SEE THE README FOR MORE DETAILS \n-----END PUBLIC KEY-----\n"
```

Comment:
maybe I'm just missing this but where does it talk about setting the private key in the new readme?

Reply:
I see that we do it in the bootstrap script but maybe we want to clarify that

Reply:
Actually, this file isn't even necessary anymore since the `bootstrap.sh` script creates it.  So I'll just remove this whole file.